src/current/v25.2/set-up-physical-cluster-replication.md

```diff
@@ -200,17 +200,6 @@ Connect to your standby cluster's system virtual cluster using [`cockroach sql`]
     --certs-dir "certs"
     ~~~
 
-1. Add your cluster organization and [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/licensing-faqs.md %}#types-of-licenses) to the cluster:
```

Comment:
Taking this out because of the licensing change. (I should go back through to v24.3 and make sure to remove.)